client/state/support/reducer.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+import {
+	SUPPORT_USER_ACTIVATED,
+	SUPPORT_USER_DEACTIVATED,
+	SUPPORT_USER_FETCH_TOKEN
+} from 'state/action-types';
+
+function isSupportUser( state = false, action ) {
```

Comment:
I like to use destructuring here because we don't really care about `action`

```js
isSupportUser( state = false, { type } ) {
	switch( type ) {
		…
	}

	return state;
}
```